sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Common/AnalyzerAdditionalFile.cs

```diff
@@ -27,6 +27,10 @@ namespace SonarAnalyzer.Common
 {
     public sealed class AnalyzerAdditionalFile : AdditionalText
     {
+
+public string field1;
+public void Unused(){}

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation. ([csharpsquid:S1186](https://sonarcloud.io/organizations/sonarsource/rules#rule_key=csharpsquid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&pullRequest=1435&issues=AWO3CDQT4427HkKVZCdr&open=AWO3CDQT4427HkKVZCdr)

[ISSUE_KEY]: AWO3CDQT4427HkKVZCdr